src/components/molecules/Visualizer/Widget/builtin.ts

```diff
@@ -2,14 +2,18 @@ import Button from "./Button";
 import Menu from "./Menu";
 import SplashScreen from "./SplashScreen";
 import Storytelling from "./Storytelling";
+import Timeline from "./Timeline";
 
 import type { Component } from ".";
 
+export const TIMELINE_BUILTIN_WIDGET_ID = "reearth/timeline";
+
 const builtin: Record<string, Component> = {
   "reearth/menu": Menu,
   "reearth/button": Button,
   "reearth/splashscreen": SplashScreen,
   "reearth/storytelling": Storytelling,
+  [TIMELINE_BUILTIN_WIDGET_ID]: Timeline,
```

Comment:
If possible I'd like to keep things consistent here. Can you export the other builtin widget ids. We aren't using them currently anywhere else, but you never know and keeps the code here more predictable.

Reply:
This is fixed by [type: fix clock type](https://github.com/reearth/reearth-web/pull/285/commits/582f1c83d5a7479b2840fcb440a185ae0ebf131a).
Thank you:pray: